lib/dpul_collections/indexing_pipeline/figgy/hydration_cache_entry.ex

```diff
@@ -88,14 +93,14 @@ defmodule DpulCollections.IndexingPipeline.Figgy.HydrationCacheEntry do
 
   defp primary_thumbnail_service_url(
          %{"thumbnail_id" => thumbnail_id} = metadata,
-         %{"member_ids" => member_data} = related_data
```

Comment:
I don't think this is a problem right now, but one potential thing with the members change is that IN() in postgres has a max of ~32,000 entries. I think in our cases that's probably fine (I can't imagine a 32k page book, and I doubt a project is going to have 32k boxless folders), but good to be aware that now there's a technical limit.

Reply:
Added note.